docs/content/concepts/scenarios/README.md

```diff
@@ -0,0 +1,13 @@
+# Document the KCP Workspace objects
+
+Document key KCP objects, with they Yaml, that are created and modified during the key steps of MC Syncer deployment:
+
+1. Creation of a KCP Workspace (`k ws create`)
+2. Creation of a KCP Syncer (`k kcp workload sync`)
+3. Deployment of the Syncer in the edge pcluster (`k apply -f`)

```

Comment:
Why edge?

We either use "physical cluster" or "workload cluster" as identifiers for the clusters running workloads - let's expand out "pcluster" to one of those to be more specific.

Reply:
Yes, this is a TMC document, let us not confuse it with edge-mc.